samcli/local/apigw/local_apigw_service.py

```diff
@@ -825,6 +825,36 @@ def _query_string_params(flask_request):
 
         return query_string_dict, multi_value_query_string_dict
 
+    @staticmethod
+    def _query_string_params_v_2_0(flask_request):
+        """
+        Constructs an APIGW equivalent query string dictionary using the 2.0 format
+
+        Parameters
+        ----------
+        flask_request request
+            Request from Flask
+
+        Returns dict (str: str)
+        -------
+            Empty dict if no query params where in the request otherwise returns a dictionary of key to value
+
+        """
+        query_string_dict = {}
+
+        # Flask returns an ImmutableMultiDict so convert to a dictionary that becomes
+        # a dict(str: list) then iterate over
+        for query_string_key, query_string_list in flask_request.args.lists():
+            query_string_value_length = len(query_string_list)
```

Comment:
it would be better to just keep the last line of L848-L854, i.e.
```
query_string_dict[query_string_key] = ",".join(query_string_list)
```
`",".join(query_string_list)` returns an empty string if `query_string_list` is an empty list.

Or, we could make the entire function even shorter:
```
query_string_dict = {
    query_string_key: ",".join(query_string_list)
    for query_string_key, query_string_list in flask_request.args.lists()
}
return query_string_dict
```